drivers/bus/acpi/acpica/include/platform/acenv.h

```diff
@@ -171,7 +171,7 @@
  *
  *****************************************************************************/
 
-#if defined(__GNUC__) && !defined(__INTEL_COMPILER)
+#if (defined(__GNUC__) || defined(__clang__)) && !defined(__INTEL_COMPILER)
```

Comment:
This is surprising, but whatever works is good. lgtm.

Reply:
This is done just to allow us to use the gcc versions until the msvc ones mature.